free-3.md

```diff
@@ -10,41 +10,100 @@ _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
 
+This language will help students see the connections between art and science,
+an hopefully stimulate people's interest in interdisciplinary boundaries.
+
+More specifically, the language will allow users to create digital graphical
+art by defining and manipulating mathematical expressions. What ContextFree is
+for grammars, this language will be for mathematics.
+
+Currently, there is a clear divide between "sciencey" disciplines and "artsy"
+disciplines, when in fact their is a great deal of creativity involved in the
+sciences (including math), and knowledge of computer science and mathematics
+can be a great help to those trying to create art. This language would attempt
+to close that divide and show users the benefits of interdisciplinary study.
```

Comment:
There are artists and groups who use mathematics as a tool for art. Many art even back in history are founded on simple mathematical concepts.